packages/frontend/app/lib/validate.server.ts

```diff
@@ -73,8 +73,7 @@ export const createAssetSchema = z
         invalid_type_error: 'Max packet amount is expected to be a number.'
       })
       .int()
-      .positive()
```

Comment:
we can have zero-decimal currencies (like JPY)

Reply:
But does `.positive()` exclude `0`? If it does, we should add a `.min(0, { message: 'Scale should be between 0 and 255' })` because isn't `int()` a signed integer?

Reply:
Good catch, I will update this